webboot/webboot.go

```diff
@@ -33,6 +33,11 @@ import (
 	"github.com/u-root/webboot/pkg/webboot"
 )
 
+const tcUrl = "http://tinycorelinux.net/10.x/x86_64/release/TinyCorePure64-10.1.iso"
```

Comment:
if you want:
const (
blah = ""
blee = ""

and so on
)
no need to repeat const. I will approve and leave it up to you.

Reply:
Done, looks much nicer indeed. :)